addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightDiscover.java

```diff
@@ -76,62 +84,62 @@ public MilightDiscover(DiscoverResult discoverResult, int resendTimeoutInMillis,
         this.discoverResult = discoverResult;
     }
 
+    /**
+     * Closes the socket and waits for the thread to shutdown.
+     * You cannot reuse this object after calling dispose.
```

Comment:
Is there a better name for a "destructor"? As far as i know, you cannot recycle a thread object, and I extend Thread, so after dispose() is called, the object can really not be used again